code-challenges.js

```diff
@@ -31,38 +44,77 @@ const temp = 42
+ //--------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
-// Describe your process:
+// Describe your process: 1.I created two variables for the two different arrays
+//2.I console log with the concat method to combine the arrays
+//3.I console log the first variable since they are combined and used the length method to show how many indexes their are in the array.
 
-const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
-const padres1998WorldSeriesRuns = [6, 3, 5, 3]
-// Expected output: 9
+let padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
+let padres1998WorldSeriesRuns = [6, 3, 5, 3]
+console.log(padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns))
+console.log(padres1998WorldSeriesRuns.length)
+//const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
+//const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+ //Expected output: 9
 
-// --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
-// Describe your process:
+ 
+ // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
+
+ //1.created a variable named currentCohort
+ //2.declared the variable as an array foxtrot 2022
+ //3.console.log the variable with the reverse 
+
+let currentCohort = ["Foxtrot 2022"]
+ console.log(currentCohort.reverse)
+
+ // Describe your process:
 
-const currentCohort = "Foxtrot 2022"
+//const currentCohort = "Foxtrot 2022"
 // Expected output: "2202 tortxoF"
 
+
+
 // --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
 
+let myNumbers = [13, 34, 42, 5, 5, 10, 27, 42, 10]
+let givenValue1 = 42
+let givenValue2 = 10
+console.log(myNumbers.lastIndexOf(42))
+console.log(myNumbers.lastIndexOf(10))
+
+//1.created a variable mynumbers for array of integers
+//2.declared variable for two given numbers
+//3.console.log my variable of arrays and used lastindex function for given value of numbers.
+
 // Describe your process:
 
-const myNumbers = [13, 34, 42, 5, 5, 10, 27, 42, 10]
-const givenValue1 = 42
-// Expected output: 7
+//const myNumbers = [13, 34, 42, 5, 5, 10, 27, 42, 10]
+// const givenValue1 = 42
+// // Expected output: 7
 
-const givenValue2 = 10
-// Expected output: 8
+// const givenValue2 = 10
+// // Expected output: 8
 
-// --------------------5) Create the code that will sort all the numbers in an array from largest to smallest. Use the test variables provided below.
+let sanDiegoSummerTemperatures = [79, 80, 72, 73, 82, 77, 76 ]
+let sanDiegoWinterTemperatures = [59, 68, 62, 59, 66, 67, 66]
+sanDiegoSummerTemperatures.sort()
+console.log(sanDiegoSummerTemperatures)
+sanDiegoWinterTemperatures.sort()
+console.log(sanDiegoWinterTemperatures)
```

Comment:
Make sure to check your code by running it in the terminal.  If it is giving you a result that you don't expect, please make sure to document that and see if you can find a solution.